### DISCUSSION TITLE
date-input-polyfill date input field visible startpage web over nginx proxy

### DISCUSSION THREAD

**Discussion by anonym-HPI** [dani-garcia/vaultwarden#1630] [Q&A]
Hi,

I am using the docker version on rasbian with nginx proxy in forground.
If I access the webpage over the http port without the proxy all seems normal, with the proxy I get the following in the left bottom corner:
![image](https://user-images.githubusercontent.com/68286419/116148914-cfdaaf80-a6e1-11eb-979a-667a6085ab69.png)
This is what is shown in dev tools in chrome:
![image](https://user-images.githubusercontent.com/68286419/116149067-fc8ec700-a6e1-11eb-8243-953809adaf41.png)

It seems new to me (never saw it), it also makes so that I can scroll a little bit on the website.

I don't know how this came. I made some nginx changes in the last days, but I reverted them.
I already removed the image and pulled it again and restarted nginx.

**Comment by BlackDex** [comment]
It looks like nginx is appending some code or something which causes this. Or maybe not sending all of the code which causes some strange things to happen.

Please checkout the https://github.com/dani-garcia/vaultwarden/wiki/Proxy-examples and compare your nginx config with the ones posted there.

Also, check you main nginx config, there could be something adding code to all you scripts.

**Reply by anonym-HPI** [reply]
I got it, I had to insert the hashes for the script for style-src.
Before it used add_header Content-Security-Policy style-src 'self', now style-src 'sha256-....' 'sha256-....', the hashes are found through the developer console in the browser.
